tests/unit/src/test/scala/tests/codeactions/StringActionsLspSuite.scala

```diff
@@ -143,6 +265,29 @@ class StringActionsLspSuite extends BaseCodeActionLspSuite("stringActions") {
        |""".stripMargin.replace("'", "\"")
   )
 
+  check(
+    "mix-strings-interpolation",
+    """|package a
+       |
+       |object A {
+       |  val e = s"hello ${} "
+       |  val c = "this <<is>> a string"
+       |  val d = s"hello ${} "
+       |}
+       |""".stripMargin,
+    s"""|${StringActions.multilineTitle}
+        |${StringActions.interpolationTitle}""".stripMargin,
+    """|package a
+       |
+       |object A {
+       |  val e = s"hello ${} "
+       |  val c = s"this is a string"
+       |  val d = s"hello ${} "
+       |}
+       |""".stripMargin,
+    1
```

Comment:
Could you use named parameter?
```suggestion
    selectedActionIndex = 1
```
best ot use it in case of primitive params, makes it clearer for the reviewers.